src/Test/Rexl.Bind.Test/Scripts/Binder/Functions/String.txt

```diff
@@ -360,3 +368,11 @@ Text.Replace("ABC", s, s)
 Text.Replace(s, s, s)
 
 Text.Replace(s, "", s)
+
```

Comment:
These tests aren't needed since they just duplicate what is above.
Tests here should be for lifting over sequence.
Also, we should add tensor based lifting tests, not just sequence.